src/components/Button/Button.tsx

```diff
@@ -22,7 +22,6 @@ export interface ButtonProps {
 	arrow?: boolean;
 	active?: boolean;
 	disabled?: boolean;
-	className?: string;
```

Comment:
deleting this will break closing of the dropdown:
https://github.com/viaacode/avo2-client/blob/feature/component-integration/src/components/CheckboxDropdown/CheckboxDropdown.tsx#L102

Reply:
will fix in client to use onOpen and onClose and keep open state outside of component